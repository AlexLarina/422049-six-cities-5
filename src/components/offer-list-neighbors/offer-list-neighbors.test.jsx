import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from 'react-router-dom';
import OfferListNeighbors from "./offer-list-neighbors.jsx";
import configureStore from "redux-mock-store";
import {Provider} from "react-redux";

const mockStore = configureStore()({
  DATA: {
    offerNeighboorsList: []
  }
});

it(`Render OfferListNeighbors`, () => {
  const tree = renderer
    .create(
        <Provider store={mockStore}>
          <MemoryRouter>
            <OfferListNeighbors
              offerNeighboorsList={[]}
            />
          </MemoryRouter>
        </Provider>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
