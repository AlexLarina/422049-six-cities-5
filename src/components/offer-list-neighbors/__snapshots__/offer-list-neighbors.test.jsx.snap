// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render OfferListNeighbors 1`] = `
<div
  className="near-places__list places__list"
/>
`;
